_includes/1.0/api/installment.html

```diff
@@ -0,0 +1,60 @@
+<div id="installment" class="group">
+
+    <h2>Installments</h2>
+    {% highlight bash %}Endpoint: {{ site.api.base_url }}{{ site.api.v1}}{{ site.resource.installments.url}}{% endhighlight %}
+    <h3 id="installmentjsonformat">JSON Format</h3>
+    <p>Installments are represented as JSON objects which have the following keys:</p>
+
+    <table class="table table-bordered table-striped">
+        <thead>
+            {% include table-header-readonly.html %}
+        </thead>
+        <tbody>
+            <tr>
+                <td>id</td>
+                <td>{{ site.format.type.integer }}</td>
+                <td>Automatically assigned.</td>
+            </tr>
+            <tr>
+                <td>invoice_id</td>
+                <td>{{ site.format.type.integer }}</td>
+                <td>Id of the related <a href="/docs/1.0/api/#invoice" title="">Invoice</a>.</td>
+            </tr>
+            <tr>
+                <td>due_date</td>
+                <td>{{ site.format.type.date }}</td>
+                <td>Date of the settlement of this installment.<br/>Format: {{ site.format.date }}<br/></td>
+            </tr>
+            <tr>
+                <td>amount</td>
+                <td>{{ site.format.type.double }}</td>
+                <td>Amount paid.</td>
+            </tr>
+            <tr>
+                <td>outstanding_amount</td>
+                <td>{{ site.format.type.double }}</td>
+                <td>Amount left to settle.</td>
+            </tr>
+            <tr>
+                <td>currency_iso_code</td>
+                <td>{{ site.format.type.string }}</td>
+                <td>The ISO code of the currency
+                    <a href="http://www.iso.org/iso/catalogue_detail?csnumber=46121" target="_blank" title="ISO 4217">(ISO 4217)</a>.
+                    4 characters max.<br />That currency should have been previously added into your company settings.<br />
+                    Default: your company's currency.
```

Comment:
Since it's a read only api, the second and third lines (4 characters max and That currency should... and default value) should be removed.